Source/RSP/RSP Command.c

```diff
@@ -29,7 +29,7 @@
 #include <stdlib.h>
 
 #include "opcode.h"
-#include "RSP.h"
+#include "Rsp.h"
 #include "CPU.h"
 #include "RSP Registers.h"
```

Comment:
@cxd4 Understood. The filename must be Rsp.h, then. I might submit a PR to "standardize" header names and includes under RSP, but it would be useful to test-compile on UNIX.

Can RSP be built out of the box with your UNIX make scripts?